python/tank/pipelineconfig.py

```diff
@@ -244,7 +244,66 @@ def __repr__(self):
 
     ########################################################################################
     # handling pipeline config metadata
-    
+
+    def get_roots_metadata_location(self):
+        """
+        Returns the location to the roots metadata.
+        """
+        return os.path.join(
+            self._pc_root, "config", "core", constants.STORAGE_ROOTS_FILE
+        )
+
+    def _get_roots_metadata(self):
```

Comment:
this was moved from pipeineconfig_utils. It is a private API and it is not being used anywhere else.